iocage.8

```diff
@@ -1791,7 +1815,7 @@ Can be set to any string if needed.
 Default: the host's release
 .Pp
 Source: local
-.It Pf compression= Op on | off | lzjb | gzip | gzip-N | zle | lz4
+.It Pf compression= Op 1 | 0 Op lzjb | gzip | gzip-N | zle | lz4
```

Comment:
Compression and dedup don’t apply with the Boolean changes

Reply:
whoops, thanks. I'll get that changed back.